### PR TITLE
[CI] Correct package name in CI test bucketing logic for accurate bucket allocation

### PR DESCRIPTION
**Pull request by sushantmane**
## Fix package name in CI test bucketing logic for proper bucket allocation
- Fix package name in CI test bucketing logic for proper bucket allocation
- Also rearranged some tests so that overall time to run E2E is below 30mins

Resolves #XXX

## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

**Review by adamxchen — DISMISSED** [review]
nice catch!

File: internal/venice-test-common/build.gradle
Comment (adamxchen) [inline internal/venice-test-common/build.gradle:1]: do you have the data for the current runtime for these?

**Review by adamxchen — APPROVED** [review]

File: internal/venice-test-common/build.gradle
Comment (sushantmane) [reply internal/venice-test-common/build.gradle:1]: The data is available, but we currently don't have an automated way to scrape it and use it to make adjustments to CI.


| JobName                      | Status     | Duration |
| ---------------------------- | ---------- | --- |
| E2ETests / IntegrationTestsA | Successful | 29m |
| E2ETests / IntegrationTestsB | Successful | 12m |
| E2ETests / IntegrationTestsC | Successful | 9m |
| E2ETests / IntegrationTestsD | Successful | 19m |
| E2ETests / IntegrationTestsE | Successful | 14m |
| E2ETests / IntegrationTestsF | Successful | 27m |
| E2ETests / IntegrationTestsG | Successful | 19m |
| E2ETests / IntegrationTestsH | Successful | 19m |
| E2ETests / IntegrationTestsI | Successful | 16m |
| E2ETests / IntegrationTestsJ | Successful | 15m |
| E2ETests / IntegrationTestsK | Successful | 20m |
| E2ETests / IntegrationTestsL | Successful | 19m |
| E2ETests / IntegrationTestsM | Successful | 17m |
| E2ETests / IntegrationTestsN | Successful | 19m |
| E2ETests / IntegrationTestsO | Successful | 21m |
| E2ETests / IntegrationTestsP | Successful | 16m |
| E2ETests / IntegrationTestsQ | Successful | 22m |
| E2ETests / IntegrationTestsZ | Successful | 29m |
| E2ETests / E2ETestsCompletion | Successful | 1m |

Comment (sushantmane) [reply internal/venice-test-common/build.gradle:1]: Maybe we should script to post these numbers post-merge on each CI run

image

**Pull request merged**